webapp-django/crashstats/crashstats/jinja2/crashstats/report_index.html

```diff
@@ -252,6 +252,14 @@
 
                             {# Crash circumstances #}
 
+                            {% if booleanish_to_boolean(raw.StartupCrash) %}
```

Comment:
I feel like I'm messing around with you when I'm not. I'm just poorly communicating it. 
What I originally wanted to say is that isn't it better to AVOID the VALUE when determining whether to show this table row. 

So, the PRESENCE of the value should determine if the table row should appear. Then within, in the `<td>` cell it should show the string "True" or "False" depending on the value. 

E.g. if `raw.StartupCrash == "1"` it should become:
```html
<tr>
  <th>Startup Crash</th>
  <td>True</td>
</tr>
```

And if `raw.StartupCrash == "0"`
```html
<tr>
  <th>Startup Crash</th>
  <td>False</td>
</tr>
```

And if `raw.StartupCrash` does not exist at all, it should simply not be any table row at all. 

What I suggest you do is keep everything as is except change this `if` statement to simply be: `{if raw.StartupCrash %}`. 

Jinja is smart and does not fail or raise an error if the key doesn't exist. In normal Python you'd get an `AttributeError` if the object `raw` doesn't have an attribute called `StartupCrash`. 
And because the value is always a non-empty string (e.g. "0" or "1") the if statement will pass if it's set to one of those and *not* pass if the key doesn't exist or is an empty string.